cryoet_data_portal_neuroglancer/state_generator.py

```diff
@@ -252,6 +252,7 @@ def combine_json_layers(
     projection_quaternion: list[float] | None = None,
     set_slices_visible_in_3d: bool | None = None,
     show_axis_lines: bool = True,
+    enable_layer_color_legend: bool = True,
```

Comment:
Looks good! We need to update the annotation shader to use the `defaultColor` instead of our custom color though, so merging this PR will have to wait a bit